apps/nextjs-website/src/components/organisms/FilteredGridLayout/FilteredGridLayout.tsx

```diff
@@ -12,21 +14,23 @@ import SectionTitle from '@/components/molecules/SectionTitle/SectionTitle';
 import { UseCase } from '@/lib/types/useCaseData';
 import { PRODUCT_HEADER_HEIGHT } from '@/components/atoms/ProductHeader/ProductHeader';
 
-type UseCaseListProps = {
-  readonly useCases: readonly UseCase[];
+type FilteredGridLayoutProps = {
+  readonly items: readonly (INewsroomItem & { tags: readonly Tag[] })[];
   readonly tags: readonly Tag[];
```

Comment:
Would put this optional
```suggestion
  readonly tags?: readonly Tag[] | null;
```